test/sharness/t0060-daemon.sh

```diff
@@ -14,6 +14,9 @@ test_expect_success "setup IPFS_PATH" '
 '
 
 # NOTE: this should remove bootstrap peers (needs a flag)
+# TODO(cryptix):
+#  - we won't see daemon startup failure because we put the daemon in the background - fix: fork with exit code after api listen
+#  - also default ports: might clash with local clients. Failure in that case isn't clear as well because pollEndpoint just uses the already running node 
```

Comment:
yeah, this issue is why iptb polls the 'ipfs id' endpoint and compares the returned ID against the expected one.